src/App.js

```diff
@@ -1,21 +1,20 @@
-import React from 'react';
-import './App.css';
+import React from "react";
```

Comment:
Why the double quotes, is that convention?

Reply:
My Prettier Plugin was not setup for this project soz....

Reply:
@clarkbab .... I notice your master branch code has some lines that end with semi colons and some dont... what gives?!